packages/ctt/src/copilot-instructions.ts

```diff
@@ -0,0 +1,131 @@
+import { Component } from 'projen';
+import { TypeScriptProject } from 'projen/lib/typescript';
+
+import {
+  CopilotInstruction,
+  CopilotInstructionOptions,
+} from './copilot-instruction';
+
+export interface CopilotInstructionsOptions {
+  /**
+   * Base path for instruction files
+   * @default '.github/instructions'
+   */
+  readonly basePath?: string;
+}
+
+export class CopilotInstructions extends Component {
+  public readonly basePath: string;
+
+  public readonly testInstruction: CopilotInstruction;
+
+  public readonly typescriptInstruction: CopilotInstruction;
+
+  private readonly instructions: Map<string, CopilotInstruction> = new Map();
+
+  constructor(
+    project: TypeScriptProject,
+    options: CopilotInstructionsOptions = {},
+  ) {
+    super(project);
+
+    this.basePath = options.basePath ?? '.github/instructions';
+
+    // Create default instructions
+    this.typescriptInstruction = this.addInstruction('typescript', {
+      applyTo: '**/*.ts,**/*.tsx',
+      content: `## Code Style
+- Use strict TypeScript configuration with all strict checks enabled
+- Prefer explicit types over \`any\` - use proper typing
+- Use optional chaining (\`?.\`) wherever possible to avoid unnecessary if statements
+- Use nullish coalescing (\`??\`) for default value assignment
+- ALWAYS prefer concise syntax like \`obj?.method()\` over verbose \`if (obj) { obj.method(); }\` unless multiple statements depend on the condition
+
+## Project Structure
+- Follow the established project structure with \`src/\` for source code
+- Place tests in \`__tests__/\` directory with \`.test.ts\` extension
+- Use proper module imports/exports
+
+## Best Practices
+- Keep functions and classes focused and single-purpose
+- Use meaningful variable and function names
+- Document complex logic with comments
+- Leverage TypeScript's type system for better code safety`,
+      name: 'TypeScript',
+    });
+
+    this.testInstruction = this.addInstruction('test', {
+      applyTo: '**/*.test.ts,**/*.spec.ts',
```

Comment:
Let's replace the spec entry with '**/__tests__/*.ts' and make it the first one.